adage/adage/settings.py

```diff
@@ -195,3 +196,7 @@
         'handlers': ['console'],
     },
 }
+
+TRIBE_ID = CONFIG['tribe_id']
+TRIBE_SECRET = CONFIG['tribe_secret']
+TRIBE_REDIRECT_URI = CONFIG['tribe_redirect_uri']
```

Comment:
Thank you for following my `CONFIG` scheme for handling secrets! Can you also update the `config.py.template` so people know what to fill in for these parameters?